src/Build/Logging/ParallelLogger/ParallelConsoleLogger.cs

```diff
@@ -535,6 +532,57 @@ public override void ProjectStartedHandler(object sender, ProjectStartedEventArg
                     WriteItems(e, e.Items);
                 }
             }
+
+            //node and project context ids for the propertyOutputMap key
+            int node = -1;
+            int projectContextId = -1;
+            if (e.BuildEventContext != null)
+            {
+                node = e.BuildEventContext.NodeId;
+                projectContextId = e.BuildEventContext.ProjectContextId;
+            }
+            //creating the value to be added to the propertyOutputMap
+            StringBuilder LogOutputProperties = new StringBuilder();
```

Comment:
Not sure of the preference in this repo, but often var is used if the RHS type is unambiguous (specifically if it is a constructor or a cast). So eg., `var logOutputProperties = new StringBuilder();`. It saves reading.

Reply:
Personally, I prefer not to use var for anything so I always know where to look if I want to know what type something is, but I know some disagree. When to use it has been on my radar for "things to talk about in a style meeting if we ever have a style meeting."

Reply:
I haven't seen var being used very often in this repo so I think I may keep it as explicitly defined